### PR TITLE
[Draft] 8290863:Update the documentation of Virtualized controls to include the best practice of not using Nodes directly in the item list

### PR DESCRIPTION
**Pull request by aghaisas**
This is a draft PR.
I have added `A warning about inserting Nodes into the ListView items list` similar to the one available for ComboBox.

I plan to update -  TreeView, TableView and TreeTableView on similar lines.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [ ] Commit message must refer to an issue
- [ ] Change requires a CSR request to be approved
- [ ] Change must be properly reviewed (2 reviews required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer), 1 [Author](https://openjdk.org/bylaws#author))

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jfx pull/867/head:pull/867` \
`$ git checkout pull/867`

Update a local copy of the PR: \
`$ git checkout pull/867` \
`$ git pull https://git.openjdk.org/jfx pull/867/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 867`

View PR using the GUI difftool: \
`$ git pr show -t 867`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jfx/pull/867.diff">https://git.openjdk.org/jfx/pull/867.diff</a>

</details>

### CODE REVIEW

**Review by kevinrushforth — COMMENTED** [review]
I think this is pretty close to what we want. A couple of comments inline.

File: modules/javafx.controls/src/main/java/javafx/scene/control/ListView.java [hunk 1]
```diff
@@ -148,6 +148,53 @@
  * ListView. See the {@link Cell} class documentation for a more complete
  * description of how to write custom Cells.
  *
+ * <h2>A warning about inserting Nodes into the ListView items list</h2>
```
[the remark this answers is not in the document]
Comment (aghaisas) [reply modules/javafx.controls/src/main/java/javafx/scene/control/ListView.java:151]: I had taken the line reference from the documentation of ComboBox.
I like your suggestion. I think we should change the warning in ComboBox documentation as well to be more in line with your suggestion.

**Comment by kevinrushforth** [comment]
This will need a CSR

/csr

**Comment by aghaisas** [comment]
New PR for openjfx20 has been created - https://github.com/openjdk/jfx/pull/995
I am closing this draft PR.

**Pull request closed without merging**